src/views/SelectJobsModal.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <ion-header>
+    <ion-toolbar>
+      <ion-buttons slot="start">
+        <ion-button @click="closeModal">
+          <ion-icon slot="icon-only" :icon="closeOutline" />
+        </ion-button>
+      </ion-buttons>
+      <ion-title>{{ $t("Select jobs") }}</ion-title>
+    </ion-toolbar>
+  </ion-header>
+  <ion-content>
+    <ion-searchbar v-model="queryString" :placeholder="$t('Search jobs')"
+      @keyup.enter="queryString = $event.target.value; search($event)" />
+
+    <ion-list v-for="job in jobs" :key="job.jobId">
```

Comment:
Add a message on UI `No jobs found`, when there is no result found for the searched string

Reply:
Done sir, through toast.